### DISCUSSION TITLE
Setting the window size in XCUITest

### DISCUSSION THREAD

**Discussion by PauliusLiekis** [pointfreeco/swift-snapshot-testing#517] [Q&A]
Hey,

I want to screenshot-snapshot my OSX application. For that I obviously need to have the same window size every time. So **how do I set the size of the main window in XCUITest**?

I understand that this is not an issue in the library, but it's very relevant to the users of this library. So maybe someone knows a solution/trick/workaround for this issue? I just don't see how can you use the library for UI testing without this...

Cheers,

**Comment by mbrandonw** [comment]
Hey @PauliusLiekis, thanks for the question! I’m not sure how to accomplish this, but I’m going to convert this issue to a discussion since it’s not a bug with the library.
